src/lib/errors.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-export function L10nError(message, id, code) {
-  this.name = 'L10nError';
-  this.message = message;
-  this.id = id;
-  this.code = code;
+export class L10nError extends Error {
+  constructor(message, id, code) {
+    this.name = 'L10nError';
```

Comment:
lint works, I built it using l20n-bundler and yes, I did run it.

The topic of super in constructor is kind of a heated one: https://esdiscuss.org/topic/super-on-class-that-extends - Gecko doesn't use it, babel requires it.